srcalc/src/main/java/gov/va/med/srcalc/web/controller/DefaultController.java

```diff
@@ -14,6 +15,13 @@
     @RequestMapping(value = "/", method = RequestMethod.GET)
     public String defaultPage()
     {
-        return "redirect:/newCalc";
+        return Views.LAUNCH_FROM_CPRS;
+    }
+
+    // This RequestMapping is hit via the spring-security configuration.
+    @RequestMapping(value = "/sessionTimeout", method = RequestMethod.GET)
```

Comment:
For now, at least. It's a configuration parameter on the server so it can easily be changed.